integration/tests/crud/creategetdelete_test.go

```diff
@@ -138,14 +145,14 @@ var _ = Describe("(Integration) Create, Get, Scale & Delete", func() {
 			Expect(config.CurrentContext).To(ContainSubstring(params.Region))
 		})
 
-		Context("and listing clusters", func() {
+		XContext("and listing clusters", func() {
```

Comment:
this looks promising 😂

Reply:
Argh :D

Reply:
I already started a full run, just came back to check it but committed the previous run's edit. :D

Reply:
@Callisto13 

Whole suite (41 tests) passed except for one which failed because it timed out.

```
2021-08-06 07:13:11 [!]  retryable error (RequestError: send request failed
caused by: Post "https://cloudformation.us-west-2.amazonaws.com/": net/http: TLS handshake timeout) from cloudformation/ListStacks - will retry after delay of 43.44878ms

• Failure [60.009 seconds]
(Integration) Create, Get, Scale & Delete
/Users/skarlso/goprojects/weaveworks/eksctl/integration/tests/crud/creategetdelete_test.go:59
  cluster with 1 node
  /Users/skarlso/goprojects/weaveworks/eksctl/integration/tests/crud/creategetdelete_test.go:126
    and listing clusters
    /Users/skarlso/goprojects/weaveworks/eksctl/integration/tests/crud/creategetdelete_test.go:148
      should return the previously created cluster [It]
      /Users/skarlso/goprojects/weaveworks/eksctl/integration/tests/crud/creategetdelete_test.go:149

      Timed out after 60.000s.
      Expected process to exit.  It did not.

      /Users/skarlso/goprojects/weaveworks/eksctl/integration/runner/runner.go:125

```

I wonder if we should do something about that. The current implementation doesn't really wait that long. Maybe we should increase the wait time on that if a retry-able error happens?